haskell/sbp.cabal.m4

```diff
@@ -1,5 +1,5 @@
 name:                  sbp
-version:               2.3.10
+version:               2.2.7
```

Comment:
@denniszollo any idea why this version number would have changed and is it important?

Reply:
@mfine will regen if this causes problems